modules/renter/contractor/editor.go

```diff
@@ -84,25 +84,38 @@ func (he *hostEditor) Upload(data []byte) (uint64, error) {
 	he.contractor.mu.RLock()
 	height := he.contractor.blockHeight
 	he.contractor.mu.RUnlock()
-	if height > he.contract.FileContract.WindowStart {
+	if height >= he.contract.FileContract.WindowStart {
```

Comment:
Do you have a test that will fail if this logic is incorrect or off-by-one? Seems like an easy mistake to make.